lang/py/avro/test/test_bench.py

```diff
@@ -22,6 +22,7 @@
 import platform
 import random
 import string
+import sys
```

Comment:
This import can be removed. (I'm never sure if everyone can see the code scanner notes, or if this comment is redundant.)

Reply:
Done. But the saga continues with a schema parser error 😄 , which seems to be transient as well.